test/componentSetup.js

```diff
@@ -18,3 +18,5 @@ export function setupComponent(jsx, options = {}) {
     ...render(jsx)
   }
 }
+
+export * from '@testing-library/react'
```

Comment:
@goosewobbler I think the changes to use `screen` throughout the tests based on the linting rules make sense, but we still want to call `setupUser()` before ever render to make sure we're using a new instance of the user object each time.

I updated this file to be more in line with the recommended way to use a setup file based on these docs:
https://testing-library.com/docs/react-testing-library/setup
https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent

Reply:
I think one more improvement would be to export the `setupComponent` method as `render` and always use that in every test as suggested in the docs. I updated this test file as a sample of how it can look when everything is put together: test/app/dash/Accounts/Add/AddPhrase/index.test.js